ASP_tests.py

```diff
@@ -323,17 +367,19 @@ def test_make_contour_grid():
     atol = 1e-9
     for wcs in [snappl.wcs.GalsimWCS.from_header(wcs_dict),
                 snappl.wcs.AstropyWCS.from_header(wcs_dict)]:
-        compare_images = np.load('tests/testdata/images.npy')
+        compare_images = np.load(pathlib.Path(__file__).parent
+                                 / 'tests/testdata/images.npy')
         image = compare_images[:11**2].reshape(11, 11)
         ra_grid, dec_grid = make_contour_grid(image, wcs)
         msg = f"RA vals do not match to {atol:.1e} using galsim wcs."
-        np.testing.assert_allclose(ra_grid[:4], test_ra, atol=atol, rtol=1e-9), msg
+        np.testing.assert_allclose(ra_grid[:4], test_ra,
+                                   atol=atol, rtol=1e-9), msg
```

Comment:
Why is `atol` specified above the for loop, but `rtol` is hardcoded here?

(Not part of this PR, just something to fix later.)